omnigibson/utils/asset_conversion_utils.py

```diff
@@ -1112,7 +1099,7 @@ def _create_urdf_import_config(
     import_config.set_merge_fixed_joints(merge_fixed_joints)
     import_config.set_convex_decomp(use_convex_decomposition)
     import_config.set_fix_base(False)
-    import_config.set_import_inertia_tensor(False)
+    import_config.set_import_inertia_tensor(True)
```

Comment:
I assume this change is intended?

Reply:
Yes, intended. This lets the moment of inertia tensors from the URDF be used.